lib/pact/command/cont_command.ex

```diff
@@ -305,7 +305,9 @@ defmodule Kadena.Pact.ContCommand do
   end
 
   @spec build_signatures(sign_commands :: sign_commands(), result :: list()) :: valid_signatures()
-  defp build_signatures([], result), do: {:ok, List.flatten(result)}
+  defp build_signatures([], result), do: {:ok, result}
+
+  defp build_signatures([%SignCommand{sig: nil} | _rest], _result), do: {:ok, []}
```

Comment:
@EdwinGuayacan What will happen to the remaining items on the list? Why return an empty list?

Reply:
There are actually no remaining elements when there are no `sig`.
change to : `defp build_signatures([%SignCommand{sig: nil}], _result), do: {:ok, []}`?